### PR TITLE
update get_token argument order to match other lib and timestamp optional

### PR DESCRIPTION
**Pull request by macphilips**
This PR introduces a breaking change and should be deployed alongside https://github.com/smileidentity/smile-identity-core-php/pull/8

### CODE REVIEW

File: lib/SmileIdentityCore.php [hunk 1]
```diff
@@ -215,23 +215,25 @@ public function get_job_status($partner_params, $options)
...
-     * @param timestamp the timestamp to generate the token from
+     * @param timestamp the iso 8601 date/time format to generate the token from
      * @param user_id
      * @param job_id
      * @param product_type - Literal value of any of the 6 product type options
      * @return array
      * @throws GuzzleException
      */
-    public function get_web_token($timestamp, $user_id, $job_id, $product_type): array
+    public function get_web_token($user_id, $job_id, $product_type, $timestamp = null, $callback_url = null): array
```
Comment (ArlonTM) [inline lib/SmileIdentityCore.php:225]: Any reason for swapping arguments/parameters?

File: lib/SmileIdentityCore.php [hunk 2]
```diff
@@ -215,23 +215,25 @@ public function get_job_status($partner_params, $options)
...
     /***
      *  Will query the backend for web session token with a specific timestamp
-     * @param timestamp the timestamp to generate the token from
+     * @param timestamp the iso 8601 date/time format to generate the token from
      * @param user_id
      * @param job_id
      * @param product_type - Literal value of any of the 6 product type options
      * @return array
      * @throws GuzzleException
      */
```
Comment (ArlonTM) [inline lib/SmileIdentityCore.php:224]: Kindly update the documentation to reflect the method's new signature/definition (ie. parameters, etc.)

File: lib/SmileIdentityCore.php (hunk 1 above)
Comment (macphilips) [reply lib/SmileIdentityCore.php:225]: Ope asks we swap it so the method signature is similar with the other libs

Comment (ArlonTM) [reply lib/SmileIdentityCore.php:226]: Okay

Comment (tamssokari) [reply lib/SmileIdentityCore.php:226]: if I recall correctly, the `get_token` method does not use a timestamp.

also, the callback_url is not optional. please adjust the function, and the comments.

**Review by ArlonTM — APPROVED** [review]
Approved!

**Pull request merged**